### PR TITLE
Add support for textarea tag as input element.

### PR DESCRIPTION
**Pull request by PharaohMaster**
Issue #383

### CODE REVIEW

**Comment by fracsi** [comment]
+1
I'd really like to see this feature.

**Comment by kevPo** [comment]
+1

File: dev/sweetalert.scss [hunk 1]
```diff
@@ -197,7 +196,14 @@ body.stop-scrolling {
       color: lighten(#575757, 40);
     }
   }
-  &.show-input input {
+  input {
+    @extend %inputCommon;
+    height: 43px;
+  }
+  textarea {
+    @extend %inputCommon;
```
Comment (kevPo) [inline dev/sweetalert.scss]: ```padding: 5px 12px;```
```resize: none;```

These might be a nice touch to the textarea too.

**Comment by thiphariel** [comment]
Hi. Consider this PR please, it's a really good feature. Thanks

**Comment by sylvainar** [comment]
+1 !

**Comment by boika** [comment]
This feature will be really useful. Thanks

**Comment by SFAntti** [comment]
+1

**Comment by y4nnick** [comment]
+1

**Comment by alex1007** [comment]
+1

**Comment by tomgoldsmith** [comment]
+1

**Comment by mattmcneeney** [comment]
+1

**Comment by kenberkeley** [comment]
use `html: true` can satisfy

http://stackoverflow.com/questions/37790965/how-to-add-textarea-tag-as-input-element-in-sweet-alert-using-jquery#answer-37791292

**Comment by t4t5** [comment]
This can now instead be done using the new [content option](https://sweetalert.js.org/docs/#content) in SweetAlert 2.0:

```javascript
swal({
  content: { 
    element: "textarea",
  } 
})
```

**Pull request closed without merging**